sdks/java/core/build.gradle

```diff
@@ -51,8 +51,6 @@ test {
 }
 
 dependencies {
-  // Required to load constants from the model, e.g. max timestamp for global window
-  shadow project(path: ":beam-model-pipeline", configuration: "shadow")
```

Comment:
Pretty sure this is the culprit.

Reply:
changed to compile, run the TextIOIT on Dataflow. Didn't help.

Reply:
I don't believe this to be the culprit.

The issue is that Dataflow has an internal container image that contains classes that were built from a previous source version. These classes become incompatible with versions submitted from "HEAD".

Reply:
Makes sense, the model project doesn't contain any of the conflicting classes. 

This should break quite often then, whenever classes are changed in `sdsks/java/core`.